### PR TITLE
Added French translations

### PR DESCRIPTION
**Pull request by FlorianLeChat**
All files with available translations have been translated into French. By the way, nice work Spar! ⭐

### CODE REVIEW

**Comment by GitSparTV** [comment]
Hi, thanks for contributing, I'm uploading all changes from all addons right now, please wait and sync

**Comment by GitSparTV** [comment]
Check again, for example Second Chance now have formatted parameter

**Comment by FlorianLeChat** [comment]
The changes have been completed, however it would be cool if the GPU saver addon could also replace the key to open the spawn menu: https://github.com/GitSparTV/GodSentTools/blob/9180de7aa1d98e6b59cf8d2b18949620e48df445/godsenttools_gpu_saver/resource/localization/en/godsenttools_gpu_saver.properties#L9
Because especially in France (and in a smaller degree in the rest of Europe), we use an AZERTY layout and the `Q` key is mostly used to move left, so most people put the bind on the `G` key or to another key. So if this key could also be replaced like you did with Second Chance, it would be really better.

**Comment by GitSparTV** [comment]
I see, I will change it to lookup binding key

**Comment by FlorianLeChat** [comment]
Everything is good on my side, I have integrated the latest changes.

File: godsenttools_gpu_saver/resource/localization/fr/godsenttools_gpu_saver.properties [hunk 1]
```diff
@@ -0,0 +1,9 @@
+godsenttools.gpusaver.name=Économiseur vidéo
+godsenttools.gpusaver.description=L'économiseur vidéo désactive entièrement le rendu lorsque la fenêtre du jeu n'est pas focalisée ou réduite. Cela permet de réduire la charge du processeur graphique et peut-être de vous épargner un crash de la mémoire virtuelle.
+godsenttools.gpusaver.osxwarning=Attention ! MacOS détecté, cet addon ne fonctionne pas avec ce système d'exploitation. L'addon sera automatiquement désactivé. Vous pouvez toujours utiliser cet addon avec le bouton « Activer manuellement ».
```
Comment (GitSparTV) [inline godsenttools_gpu_saver/resource/localization/fr/godsenttools_gpu_saver.properties:3]: Do you really put spaces before `!` and `:` in your language?
Same question about spaces between « »

**Review by GitSparTV — CHANGES_REQUESTED** [review]

File: godsenttools_gpu_saver/resource/localization/fr/godsenttools_gpu_saver.properties (hunk 1 above)
Comment (FlorianLeChat) [reply godsenttools_gpu_saver/resource/localization/fr/godsenttools_gpu_saver.properties:3]: Yes, this is a specification of the French language: https://en.wikipedia.org/wiki/History_of_sentence_spacing#French_and_English_spacing

**Comment by GitSparTV** [comment]
Alright, I've updated files again little bit.

**Review by GitSparTV — APPROVED** [review]

**Comment by FlorianLeChat** [comment]
I will wait a few days before updating my pull request again, so that you can make any necessary changes during this time. If you think you're not going to touch the translations anymore, let me know!

**Comment by GitSparTV** [comment]
I'm done :)

**Comment by FlorianLeChat** [comment]
Here we go. I also fixed two small inconsistencies in the English translations of the "Pitch Black" addon. Let me know if everything is good for you!

**Review by GitSparTV — APPROVED** [review]

**Comment by GitSparTV** [comment]
Translations are uploaded to Workshop, everything works for me. Thanks for contribution

**Pull request merged**